test/index.js

```diff
@@ -366,36 +398,77 @@ t.test('prompt, refuses', async t => {
   const runPath = path
   const cache = resolve(testdir, 'cache')
   const npxCache = resolve(testdir, 'npxCache')
-  const libexec = t.mock('../lib/index.js', {
-    '@npmcli/ci-detect': () => false,
-    read (opts, cb) {
-      cb(null, 'n')
-    },
-    '../lib/no-tty.js': () => false,
+  t.test('with clearProgress function', async t => {
+    const libexec = t.mock('../lib/index.js', {
+      '@npmcli/ci-detect': () => false,
+      'proc-log': {
+        clearProgress () {
+          t.ok(true, 'should call clearProgress function')
+        }
+      },
+      read (opts, cb) {
+        cb(null, 'n')
+      },
+      '../lib/no-tty.js': () => false,
+    })
+
+    await t.rejects(
+      libexec({
+        ...baseOpts,
+        args: ['@ruyadorno/create-index'],
+        cache,
+        npxCache,
+        path,
+        runPath,
+        yes: undefined,
+      }),
+      /canceled/,
+      'should throw with canceled error'
+    )
+
+    const installedDir = resolve(npxCache,
+      '0e8e15840a234288/node_modules/@ruyadorno/create-index/package.json')
+
+    t.throws(
+      () => fs.statSync(installedDir),
+      { code: 'ENOENT' },
+      'should not have installed required packages'
+    )
   })
 
-  await t.rejects(
-    libexec({
-      ...baseOpts,
-      args: ['@ruyadorno/create-index'],
-      cache,
-      npxCache,
-      path,
-      runPath,
-      yes: undefined,
-    }),
-    /canceled/,
-    'should throw with canceled error'
-  )
+  t.test('with clearProgress function', async t => {
```

Comment:
I think this was meant to say `with no clearProgress function`?

Reply:
Oh yeah, that's awkward.. Updated